src/model/config.ts

```diff
@@ -85,19 +62,10 @@ const DEFAULT_CONFIG: RainbowConfig = {
     base: 200,
     zora: 200,
   }),
-  ethereum_goerli_rpc: __DEV__ ? ETHEREUM_GOERLI_RPC_DEV : ETHEREUM_GOERLI_RPC,
-  ethereum_mainnet_rpc: __DEV__
-    ? ETHEREUM_MAINNET_RPC_DEV
-    : ETHEREUM_MAINNET_RPC,
   f2c_enabled: true,
   flashbots_enabled: true,
   op_nft_network: 'op-mainnet',
   op_rewards_enabled: false,
-  optimism_mainnet_rpc: OPTIMISM_MAINNET_RPC,
```

Comment:
if were gonna have this remotely configured it would make sense to keep these for a fallback i think